backend/src/main.py

```
@@ -81,12 +70,66 @@ def internal_error(error):
+    global background_loop
+    background_loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(background_loop)
+
+    # Schedule background tasks
+    # Access properties to initialize them within the loop context
+    _ = data_fetcher_service.http_client
+    _ = ai_analysis_service.http_client
+    _ = trading_service.solana_client
+    _ = trading_service.http_client
+
+    background_loop.create_task(mempool_monitor_service.start_monitoring())
```

Ruff (0.15.9)

[warning] Store a reference to the return value of `background_loop.create_task`

(RUF006)